### ISSUE TITLE
Ask for adding: CAPSImg_5.1.2_Linux_ARM64.tar.xz

### ISSUE THREAD

**Issue by benas80** [FrodeSolheim/capsimg#1]
Could i ask You for add: CAPSImg_5.1.2_Linux_ARM64.tar.xz

(ARM64 version of this "capsimg.so" for linux)

Thank You very much.

**Comment by MichaIng** [comment]
[CAPSImg_5.1.2_Linux_ARMv8.tar.xz](https://github.com/FrodeSolheim/capsimg/releases/download/v5.1.2/CAPSImg_5.1.2_Linux_ARMv8.tar.xz) should be what you are looking for: ARMv8 == ARM64.

**Comment by FrodeSolheim** [comment]
No, the ARMv8 build is in fact 32-bit. ARMv8 CPUs are the first which (optionally) supports 64-bit, but this build is intended for 32-bit operating systems such as (the 32-bit) Raspberry Pi OS on Raspberry Pi 4.

**Comment by MichaIng** [comment]
Ah sorry, my fault then.

As it's 32-bit, it indeed works well on ARMv7 (just tested it on Raspberry Pi 2 🙂). It seems to have been compiled with NEON, respectively with Debian armhf development tools/libs, instead of Raspbian one, otherwise it would work on ARMv6 as well, but it fails on my Raspberry Pi Zero W.

Hence I suggest to rename it to `CAPSImg_5.1.2_Linux_ARMv7.tar.xz` to better indicate the minimum hardware and OS arch requirements. That ARMv7/64-bit is backwards-compatible, is clear, I think.
